tests/scripts/thread-cert/border_router/MATN_05_ReregistrationToSameMulticastGroup.py

```diff
@@ -0,0 +1,244 @@
+#    ----------------(eth)------------------
+#           |                  |     |
+#          BR_1 (Leader) ---- BR_2  HOST
+#           |                  |
+#           |                  |
+#          Router_1 -----------+
+#
+
+BR_1 = 1
+BR_2 = 2
+ROUTER_1 = 3
+HOST = 4
+
+REG_DELAY = 10
+UDP_HEADER_LENGTH = 8
+
+
+class MATN_05_ReregistrationToSameMulticastGroup(thread_cert.TestCase):
+    USE_MESSAGE_FACTORY = False
+
+    TOPOLOGY = {
+        BR_1: {
+            'name': 'BR_1',
+            'is_otbr': True,
+            'allowlist': [BR_2, ROUTER_1],
+            'version': '1.2',
+            'router_selection_jitter': 2,
+        },
+        BR_2: {
+            'name': 'BR_2',
+            'allowlist': [BR_1, ROUTER_1],
+            'is_otbr': True,
+            'version': '1.2',
+            'router_selection_jitter': 2,
+        },
+        ROUTER_1: {
+            'name': 'Router_1',
+            'allowlist': [BR_1, BR_2],
+            'version': '1.2',
+            'router_selection_jitter': 2,
+        },
+        HOST: {
+            'name': 'Host',
+            'is_host': True
+        },
+    }
+
+    def test(self):
+        br1 = self.nodes[BR_1]
+        br2 = self.nodes[BR_2]
+        router1 = self.nodes[ROUTER_1]
+        host = self.nodes[HOST]
+
+        br1.set_backbone_router(reg_delay=REG_DELAY, mlr_timeout=consts.MLR_TIMEOUT_MIN)
+        br1.start()
+        self.simulator.go(10)
+        self.assertEqual('leader', br1.get_state())
+        self.assertTrue(br1.is_primary_backbone_router)
+
+        router1.start()
+        self.simulator.go(10)
+        self.assertEqual('router', router1.get_state())
+
+        br2.start()
+        self.simulator.go(10)
+        self.assertEqual('router', br2.get_state())
+        self.assertFalse(br2.is_primary_backbone_router)
+
+        host.start(start_radvd=False)
+        self.simulator.go(10)
+
+        # Router_1 registers for multicast address, MA1, at BR_1.
+        router1.add_ipmaddr(MA1)
+        self.simulator.go(5)
+
+        # 1. Host sends a ping packet to the multicast address, MA1.
+        self.assertTrue(
+            host.ping(MA1, backbone=True, ttl=10, interface=host.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
+
+        # Ensure Router_1 renews its multicast registration
+        self.simulator.go(consts.MLR_TIMEOUT_MIN - 10)
+
+        # 4. Within MLR_TIMEOUT_MIN seconds, Host sends a ping packet to the
+        # multicast address, MA1. The destination port 5683 is used for the UDP
+        # Multicast packet transmission.
+        host.udp_send_host(data='PING', ipaddr=MA1, port=5683)
+        self.simulator.go(5)
+
+        # 6a. By internal means, Router_1 stops listening to the multicast
+        # address, MA1.
+        router1.del_ipmaddr(MA1)
+
+        # 7. After (MLR_TIMEOUT_MIN+2) seconds, Host multicasts a ping packet to
+        # multicast address, MA1, on the backbone link.
+        self.simulator.go(consts.MLR_TIMEOUT_MIN + 4)
```

Comment:
```suggestion
        # 7. After (MLR_TIMEOUT_MIN+2) seconds, Host multicasts a ping packet to
        # multicast address, MA1, on the backbone link.
        self.simulator.go(consts.MLR_TIMEOUT_MIN + 2)
```

The code is not inline with the comment.